docs/ExpectAPI.md

```diff
@@ -720,7 +720,7 @@ test('drinkEach drinks each drink', () => {
 
 Also under the alias: `.toBeCalledWith()`
 
-Use `.toHaveBeenCalledWith` to ensure that a mock function was called with specific arguments.
+Use `.toHaveBeenCalledWith` to ensure that a mock function was called with specific arguments.The specific arguments are checked with a `.toEqual` call.
```

Comment:
```suggestion
Use `.toHaveBeenCalledWith` to ensure that a mock function was called with specific arguments. The specific arguments are checked with a `.toEqual` call.
```

same for the others

Reply:
also, please use the wording I provided in https://github.com/facebook/jest/pull/12222#discussion_r800033248